### PR TITLE
[TASK][CERT-125] Remove readonly property on datepicker fields

### PR DESCRIPTION
**Pull request by tomasnorre**

### CODE REVIEW

**Comment by tomasnorre** [comment]
OK. Then we misunderstood each other completely. I'll look at it again.

**Review by benjaminkott — CHANGES_REQUESTED** [review]
Please make sure to reset all overwritten properties

https://github.com/twbs/bootstrap/blob/v5.1.3/scss/forms/_form-control.scss#L69-L72

**Comment by tomasnorre** [comment]
I'm unsure regarding the `opacity: 1` as `1` is the default value for opacity. 
Please let me know if I should change that too. 

https://github.com/twbs/bootstrap/blob/v5.1.3/scss/forms/_form-control.scss#L69-L72

**Review by benjaminkott — CHANGES_REQUESTED** [review]
frocus border is still mismatching

File: assets/css/components/_datepicker.scss [hunk 1]
```diff
@@ -31,3 +31,10 @@
         display: block;
     }
 }
+// Reset styling for flatpick-input[readonly] to remove the "readonly"-styling
+.flatpickr-input[readonly] {
```
Comment (benjaminkott) [inline assets/css/components/_datepicker.scss:35]: try `.flatpickr-input[readonly]:not([disabled])`

Comment (tomasnorre) [reply assets/css/components/_datepicker.scss:35]: That works too.

**Review by benjaminkott — APPROVED** [review]

**Pull request merged**